_posts/2020-10-15-2.0.0-release.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Apache Arrow 2.0.0 Release"
+date: "2020-10-14 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+<!--
+
+To use this template:
+
+* Update all "XX" values with the appropriate numbers (you can get the resolved issues and contributors count from `_release/2.0.0.md`)
+* Fill in the various sections below. Note that the audience is the broader user community, not Arrow developers, so please write clearly using terms they will understand and care about. Delete any sections that don't have any content (as in, there are no changes to announce)
+* Delete this introductory comment
+
+ -->
+
+
+The Apache Arrow team is pleased to announce the 2.0.0 release. This covers
+over XX months of development work and includes [**XX resolved issues**][1]
+from [**XX distinct contributors**][2]. See the Install Page to learn how to
```

Comment:
@kszucs can you fill these numbers in now?

Reply:
Available in the [release note pr](https://github.com/apache/arrow-site/pull/80/files#diff-0f3f18b3362ba5c10197b69c36d78008aa794c049f82b015f89ce282a37dc927R41) and [JIRA release page](https://issues.apache.org/jira/projects/ARROW/versions/12345977).